src/2_branch.js

```diff
@@ -74,16 +73,40 @@ Branch.prototype['init'] = function(app_id, callback) {
  * @param {function|null} callback
  */
 Branch.prototype['logout'] = function(callback) {
+	callback = callback || function() {};
 	if (!this.initialized) { return callback(utils.message(utils.messages.nonInit)); }
+	var self = this;
+	this.api(resources.logout, {
+		session_id: this.session_id,
+		app_id: this.app_id
+	}, function(err, data) {
+		if (err) { callback(err); }
+		else {
+			self.session_id = data['session_id'];
+			self.identity_id = data['identity_id'];
+			utils.store(data);
+			callback(data);
+		}
+	});
+};
+
+/**
+ * @param {function|null} callback
+ */
+Branch.prototype['close'] = function(callback) {
```

Comment:
Agreed, I can't see any possible reason for needing to do this. I haven't done anything with it yet though. Perhaps get some input from a few others? There just may be edge cases I'm not aware of.